boards/arm/beagleconnect_freedom_cc1352r/Kconfig.board

```diff
@@ -0,0 +1,9 @@
+# TI CC1352R LaunchXL board
```

Comment:
This is not  a "LaunchXL" board. It is also not a TI board. It also uses CC1352P, not CC1352$.

You should base this off of https://github.com/jadonk/beagle_connect_zephyr_board/blob/93a5a3f96b0c8b47634218c9d7a40741459c316d/Kconfig.board and be sure to credit Erik Larson who did the initial hardware bring-up!